packages/report-server/src/__tests__/reportBuilder/output/output.test.ts

```diff
@@ -116,6 +116,29 @@ describe('output', () => {
     });
 
     describe('handle categoryField', () => {
+      it('throws error if categoryField matches any columns', () => {
+        const output = buildOutput({
+          type: 'matrix',
+          categoryField: 'InfrastructureType',
+          rowField: 'FacilityType',
+          columns: ['InfrastructureType', 'Laos', 'Tonga'],
+        });
+        expect(() => {
+          output(MULTIPLE_TRANSFORMED_DATA_WITH_CATEGORIES);
+        }).toThrow();
```

Comment:
I like to specify the message we're expecting, so we don't get false negatives if it starts throwing something else like undefined is not a function or whatever.

Reply:
I did think of this, and given you feel that way too I'm happy to update these tests. Sometimes I worry it makes the tests a bit too brittle, but it's probably worth it especially since these can be user facing errors